packages/visual-editor/src/vite-plugin/templates/main.tsx

```diff
@@ -90,6 +91,10 @@ export const getPath: GetPath<TemplateProps> = ({ document }) => {
 
 const Location: Template<TemplateRenderProps> = (props) => {
   const { document } = props;
+  const filteredConfig = filterComponentsFromConfig(
```

Comment:
Noting that you'll need to add this to the starter too

Reply:
My mistake, we don't have the templates in the starter anymore, only the default data. Nvm then!